### PR TITLE
Summary option.

### PR DESCRIPTION
**Pull request by skempdt**
Add option to just show number of matching entries, rather than details about each.

### CODE REVIEW

File: man/dnstable_lookup.1 [hunk 1]
```diff
@@ -113,6 +113,11 @@ presentation format\&. One JSON\-formatted entry per line will be printed\&.
 When using a JSON format for output, add a hex representation of the raw rdata and the rrtype\&.
 .RE
 .PP
+\fB\-s\fR
```
Comment (cmikk) [inline man/dnstable_lookup.1:116]: Option needs to be updated in the manual page, also.

File: src/dnstable_lookup.c [hunk 2]
```diff
@@ -32,6 +32,7 @@
 static bool g_json = false;
 static bool g_Json = false;
 static bool g_add_raw;
+static bool g_summary = false;
```
Comment (cmikk) [inline src/dnstable_lookup.c:35]: Consider renaming this to `g_count` or similar. We may wish to add a full summary along the lines of `dnsdbq -V summarize` in the future.

**Review by cmikk — CHANGES_REQUESTED** [review]
Requesting two minor cleanup changes, otherwise lgtm.

File: man/dnstable_lookup.1 (hunk 1 above)
Comment (skempdt) [reply man/dnstable_lookup.1]: Done. I note the newly added -C case insensitive is missing in the man-page and corresponding .txt file.

File: src/dnstable_lookup.c (hunk 2 above)
Comment (skempdt) [reply src/dnstable_lookup.c]: Done.

File: src/dnstable_lookup.c [hunk 3]
```diff
@@ -236,6 +245,9 @@ main(int argc, char **argv)
 		case 'J':
 			g_Json = true;
 			break;
+		case 'n':
```
Comment (shw700) [inline src/dnstable_lookup.c:248]: Is missing corresponding change to getopt()

Comment (skempdt) [reply src/dnstable_lookup.c:248]: Fixed.

**Review by shw700 — APPROVED** [review]

**Pull request merged**